slides/k8s/csr-api.md

```diff
@@ -60,7 +60,7 @@ This is what I do if I want to obtain a certificate.
 
 1. Create public and private key.
 
-2. Create a Certificate Signing Request (CSR).
+2. Create a Certificate Signing Request (CSR) using this private key.
```

Comment:
I'm going to revert that specific line, since the CSR isn't really created "using this private key". Everything else is :100: